test/repl.jl

```diff
@@ -167,6 +167,21 @@ fake_repl() do stdin_write, stdout_read, repl
     close(t)
     readuntil(stdout_read, "\n\n")
 
+    # Test that module prefix is omitted when type is reachable from Main
```

Comment:
I couldn't find a simpler way to test `display(::REPLDisplay, ...)`. If there's a cleaner solution, I'm all ears.

Reply:
Well, as I said, you can

> eval [...] in `Main`

Reply:
But eval what? :-)

Reply:
Whatever you need to be run in `Main`?

Instead of `write(stdin_write, "module TestShowTypeREPL; export TypeA; struct TypeA end; end\n")` and `write(stdin_write, "using .TestShowTypeREPL\n")` just do `eval` instead?

`@eval Main module TestShowTypeREPL; export TypeA; struct TypeA end; end` and `@eval Main using .TestShowTypeREPL`

Reply:
Thanks, that makes the code simpler. Though I haven't found a way to get the result without the whole `readline` stuff.